### ISSUE TITLE
[Feature]judge today, yesterday, tomorrow methods

### ISSUE THREAD

**Issue by shinriyo** [jogboms/time.dart#30]
Do you want these?

If so, I will pull-request.

```
someDateTimeValue.isToday();

=> true or false
```

```
someDateTimeValue.isTomorrow();

=> true or false
```

```
someDateTimeValue.isYesterday();

=> true or false
```

**Comment by Katekko** [comment]
> Do you want these?
> 
> If so, I will pull-request.
> 
> ```
> someDateTimeValue.isToday();
> 
> => true or false
> ```
> 
> ```
> someDateTimeValue.isTomorrow();
> 
> => true or false
> ```
> 
> ```
> someDateTimeValue.isYesterday();
> 
> => true or false
> ```

You can do without (), use like a GET attribute:  `date.isYesterday`


Instead you create a method like:

```
bool isYesterday(){
  return true;
}
```

you will create:

`bool get isYesterday => return true;`

or 

```
bool get isYesterday {
  return true;
}
```

**Comment by shinriyo** [comment]
@Katekko Hello. Do you want this?

p.s. I like Courage the Cowardly Dog!

**Comment by shinriyo** [comment]
I did PR

[Feature]#30

**Comment by Ascenio** [comment]
What do you about changing `isYesterday` to `wasYesterday`?

**Comment by shinriyo** [comment]
@Ascenio I see. It is past tense. I didn't think about tense.

**Comment by shinriyo** [comment]
@Ascenio I fix it in PR.

@jogboms
Can you check?

**Comment by jogboms** [comment]
Closed by #32

**Comment by shinriyo** [comment]
thanks
